purchase_stock_price_variance/readme/CONFIGURE.md

```diff
@@ -0,0 +1,15 @@
+1. Navigate to Inventory > Configuration > Settings.
+2. Find and enable the "Enable Price Variance Threshold Check" option.
+3. Set the following global values to apply if no specific value is set at the product level.
+    If the threshold value is set to 0, this threshold will not be checked.:
+    * Price Variance Threshold Percent – Default percentage variance for all products.
+    * Price Variance Threshold Amount – Default maximum variance for all products.
+4. Go to Inventory > Products and open the product.
+5. Click on the Inventory tab and configure the following fields.
+    If the threshold value is set to 0, the threshold will refer to the global value.
+    It is not possible to disregard a single global threshold value from the product level.:
+    * Bypass Price Variance Check – Enable this to skip the check for this specific product.
+    * Price Variance Threshold Percent – Set the allowable percentage variance.
+    * Price Variance Threshold Amount – Define the maximum allowable price variance.
```

Comment:
We should either add more explanation on the "limitation" with an example, or remove the sentence to avoid confusing the audience.
```suggestion
    * **Bypass Price Variance Check**: Enable this to skip the check for this specific product.
    * **Price Variance Threshold Percent**: Set the allowable percentage variance.
    * **Price Variance Threshold Amount**: Define the maximum allowable price variance.
```